tests/unit/amo/components/TestCollection.js

```diff
@@ -432,69 +449,72 @@ describe(__filename, () => {
       }),
     );
 
-    const page = 123;
-    const location = fakeRouterLocation();
-    const newLocation = fakeRouterLocation({ query: { page } });
     const errorHandler = createStubErrorHandler();
 
     const wrapper = renderComponent({
       errorHandler,
-      location,
+      location: fakeRouterLocation({ query: filters }),
       store,
     });
     fakeDispatch.resetHistory();
 
+    const newFilters = {
+      ...filters,
+      page: 999,
+    };
+
     // This will trigger the componentWillReceiveProps() method.
-    wrapper.setProps({ location: newLocation });
+    wrapper.setProps({ filters: newFilters });
 
     sinon.assert.callCount(fakeDispatch, 1);
     sinon.assert.calledWith(
       fakeDispatch,
       fetchCurrentCollectionPage({
         errorHandlerId: errorHandler.id,
-        page,
+        filters: newFilters,
         username: defaultUser,
         slug: defaultSlug,
       }),
     );
   });
 
-  it('defaults to first page when there is no page and collection is loaded', () => {
```

Comment:
This test case was removed because there is no longer a condition where the component is instantiated without a `page`. `mapStateToProps` now generates the `filters` prop and it defaults the `page` filter to `1`, which is tested in the test case for `it('dispatches fetchCurrentCollection on mount')`. I can create a separate test case to assert the default values for `filters`, if you prefer, but it seemed unnecessary as they're tested in that case.